documentation/_include/virtualization.adoc

```diff
@@ -103,7 +103,7 @@ will not work, but it does mean that (if it does work) it will be VERY slow.
 
 NOTE: Please note that all software has been pre-installed and configured.  These steps are provided as reference material only.
 
-System needs to be configure with access to the following repos:
+The system needs configured with access to the following repos:
```

Comment:
The system needs to be configured with access to the following repos:

Reply:
I've read it both ways and I think that "to be" could be dropped before the past tense configured and it still provide the same message.